tests/fixture_island_nesting/main.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference no-default-lib="true" />
```

Comment:
I feel like we'll need to improve the test fixture situation here soon. Unfortunate that adding new tests adds so much bloat

Reply:
Yeah true :S On the hand it's kinda neat to just go in a directory and boot up a test fixture to hack on without the test runner around it though. The verbosity of creating these fixtures though is a bit much nonetheless :S